lib/datadog/lambda/trace/constants.rb

```diff
@@ -20,11 +20,16 @@ module Trace
     DD_XRAY_SUBSEGMENT_NAME = 'datadog-metadata'
     DD_XRAY_SUBSEGMENT_KEY = 'trace'
     DD_XRAY_SUBSEGMENT_NAMESPACE = 'datadog'
+    DD_TRACE_MANAGED_SERVICES = 'DD_TRACE_MANAGED_SERVICES'
     SOURCE_XRAY = 'XRAY'
     SOURCE_EVENT = 'EVENT'
     XRAY_ENV_VAR = '_X_AMZN_TRACE_ID'
     XRAY_UDP_PORT = 2000
     LOCAL_HOST = '127.0.0.1'
     AWS_XRAY_DAEMON_ADDRESS_ENV_VAR = 'AWS_XRAY_DAEMON_ADDRESS'
+
+    # Header that prevents dd-trace-rb Net::HTTP integration from tracing
+    # internal trace requests. Set it to any value to skip tracing.
+    HEADER_DD_INTERNAL_UNTRACED_REQUEST = 'DD-Internal-Untraced-Request'
```

Comment:
Maybe instead of defining it, we could use the one in the tracer so we don't use something tracer-internal?

Reply:
Refactored.